### DISCUSSION TITLE
gcp/2020/04/12/bigquery-unnest-array-struct/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#122] [Comments]
#  BigQuery UNNEST, ARRAY, STRUCT 사용 방법

https://zzsza.github.io/gcp/2020/04/12/bigquery-unnest-array-struct/

**Comment by zzsza** [comment]
<p>감사합니다 :) 덕분에 unnest 개념 바로 이해했습니다!</p>

>_This comment was made by Disqus. 2020-12-27 10:22:51 Jiwhan Kim_

**Comment by jackson-jspark** [comment]
덕분에 unnest에 대해서 확실하게 알게 되었고, 현업에서 실제로 사용하여 원하던 결과를 간편하게 구현하였습니다.
감사합니다!
